microservices/notification/src/jobs/task/process.ts

```diff
@@ -0,0 +1,23 @@
+import { Endpoint } from '@lomray/microservice-helpers';
+import { IsNumber } from 'class-validator';
+import ProcessService from '@services/task/process';
+
+class ProcessOutput {
+  @IsNumber()
+  count: number;
+}
+
+/**
+ * Process tasks
+ */
+const process = Endpoint.custom(
+  () => ({
+    output: ProcessOutput,
+    description: 'Process tasks',
+  }),
+  async () => ({
+    count: await ProcessService.init().checkoutAndProcess(),
```

Comment:
```suggestion
    count: await ProcessService.init().checkoutAndProcess(),
```
checkout - related with payments word, change